planning/behavior_velocity_planner/autoware_behavior_velocity_detection_area_module/README.md

```diff
@@ -114,3 +116,9 @@ This module has parameter `hold_stop_margin_distance` in order to prevent from t
   ![example](docs/keep_stopping.svg){width=1000}
   <figcaption>inside the hold_stop_margin_distance</figcaption>
 </figure>
+
+#### Feasible stop distance
+
+If `use_max_acceleration` is _true_, the module ensures the vehicle can stop within the physical limit set by `max_acceleration`.
```

Comment:
@yhisaki 
The stopping points are inserted by considering the braking distance calculated from max_acceleration, and the module does not ensure that the vehicle can actually stop.